aws_helper/config.go

```diff
@@ -28,3 +29,26 @@ func CreateAwsSession(awsRegion, awsProfile string) (*session.Session, error) {
 
 	return session, nil
 }
+
+// AssumeRoleEnvironmentVariables returns a set of key value pair to use as environment variables to assume a different role
+func AssumeRoleEnvironmentVariables(roleArn string, sessionName string) (map[string]string, error) {
+	session, err := CreateAwsSession("", "")
```

Comment:
In fact, that will work as is. the CreateAwsSession will create a session with the assumed role developer in account B and the AssumeroleEnvironmentVariables function will return the env vars required to assume the deployment role in B.